packages/opentelemetry-core/src/platform/BaseAbstractPlugin.ts

```diff
@@ -27,7 +27,6 @@ export abstract class BaseAbstractPlugin<T> implements Plugin<T> {
   protected _config!: PluginConfig;
   protected _internalFilesExports!: { [module: string]: unknown }; // output for internalFilesExports
   protected readonly _internalFilesList?: PluginInternalFiles; // required for internalFilesExports
-  protected _logger!: Logger;
```

Comment:
I thought this was going to point to `api.diag` so we could prevent breaking existing plugins